deploy-manage/security/secure-settings.md

```diff
@@ -260,23 +204,24 @@ There are reloadable secure settings for:
 * [LDAP realm](elasticsearch://reference/elasticsearch/configuration-reference/security-settings.md#ref-ldap-settings)
 * [Remote cluster credentials for the API key based security model](/deploy-manage/remote-clusters/remote-clusters-settings.md#remote-cluster-credentials-setting)
```

Comment:
I think we should kill this list ... what do you think? this feels to me like a possible maintenance risk (people might add more reloadable settings without editing this list)

imo the person is going to be looking for settings first, and reloading them as described here as needed (instead of seeking out reloadable settings)

Reply:
done!